metron-deployment/inventory/full-dev-platform/group_vars/all

```diff
@@ -78,7 +79,7 @@ sensor_test_mode: True
 install_pycapa: False
 install_bro: True
 install_snort: True
-install_yaf: True
+install_yaf: False
```

Comment:
If you just want to not start YAF, then not having it in the 'services_to_start' list should be sufficient.  The change here causes YAF not to be installed.

On a side note, I would expect Snort and Bro to be much larger resource hogs than YAF.